edge/Dockerfile

```diff
@@ -11,32 +11,39 @@ ARG ARCH=x64
 ARG REDISAI_VERSION=1.0.2
 ARG REDISTIMESERIES_VERSION=1.4.6
 ARG REDISGEARS_VERSION=1.0.2
-ARG REDIS_VER=6.0.9
+ARG REDIS_VERSION=6.0.9
```

Comment:
`docker.native` uses `REDIS_VER`.

Reply:
Gah - I changed it to REDIS_VERSION throughout because it was prettier. *sighs and changed*